libraries/Microsoft.Bot.Builder.AI.QnA/Schemas/Microsoft.QnAMakerDialog.schema

```diff
@@ -112,6 +112,25 @@
                     "$ref": "schema:#/definitions/equalsExpression"
                 }
             ]
+        },
+        "strictFiltersCompoundOperationType": {
+            "$ref": "schema:#/definitions/stringExpression",
+            "title": "StrictFilters Compound Operation Type",
+            "description": "Type of Operation for Strict Filters.",
+            "oneOf": [
+                {
+                    "title": "StrictFiltersCompoundOperationType",
+                    "description": "StrictFiltersCompoundOperationType.",
+                    "enum": [
+                        "OR",
+                        "AND"
```

Comment:
the order is AND, OR right?, please keep it consistent (as that has impact on 0, 1 value)

Reply:
implemented the change.

---
In reply to: [470570382](https://github.com/microsoft/botbuilder-dotnet/pull/4446#discussion_r470570382) [](ancestors = 470570382)